src/color/tier.json

```diff
@@ -81,6 +81,22 @@
               "wcag": "n/a",
               "deprecated": false
             }
+          },
+          "lounge": {
+            "value": "{color.brand.midnight.400.value}",
+            "public": true,
+            "onDark": true,
```

Comment:
```suggestion
```

The `onDark` attribute is deprecated and SHOULD NOT be used on any new tokens. If the color is to be used on darker backgrounds, the new concept is `inverse`. 

I am assuming that this was copied for a previous token and left in.

Reply:
I've resolved your feedback. Please revalidate the changes. Thank you